runtime/src/main/java/com/gabrielittner/threetenbp/GeneratedZoneIdsProvider.java

```diff
@@ -549,6 +550,7 @@ final class GeneratedZoneIdsProvider implements ZoneIdsProvider {
       "Pacific/Guam",
       "Pacific/Honolulu",
       "Pacific/Johnston",
+      "Pacific/Kanton",
```

Comment:
This one has been included prior to `2022b`. But we never update it before. Hence, now it is being added up.